polaris-ui/src/app/features/cases/presentation/case-details/utils/checkMultipleDefendantsOrCharges.ts

```diff
@@ -0,0 +1,16 @@
+import { CaseDetails } from "../../../domain/CaseDetails";
+
+export const checkMultipleDefendantsOrCharges = (
+  caseDetails: CaseDetails
+): boolean => {
+  const { defendants } = caseDetails;
+  if (defendants.length > 1) {
+    return true;
+  }
+
+  const { charges } = defendants[0];
```

Comment:
Would this blow up if the defendant array is empty? (Especially in cin3, we see case data that is not particularly realistic or well-formed, so guess we could see empty defendants).

Also, depending on your style preference, we could do something like call this util function `isSingleChargeCase` and then  have a one-liner `return defendants.length === 1 && defendants[0].charges.length === 1`